app/components/ProductList/productItem.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import {Router, Route, Link, IndexRoute, hashHistory} from 'react-router'
+
 import styles from './products.css'
 
 
-const Item = ({ id, imageUrl, name, price }) => (
-  <div className={styles.item} >
-    <img className="image" src={imageUrl} alt=""/>
-    <div className={styles.info}>
-      <div className={styles.id}>{id}</div>
-      <div className={styles.title}>{name}</div>
-      <div className={styles.price}>{price}</div>
+const Item = ({ id, imageUrl, name, price }) => {
```

Comment:
Good usage of functional component. You don't need the curly braces here. Just doing

```
const Item = ({ id, imageUrl, name, price }) => (
<div>
...
);
```

is enough